db/seeds/development.rb

```diff
@@ -40,3 +39,13 @@
                access_token_audience: 'va.gov',
                logout_redirect_uri: 'http://localhost:3001',
                refresh_token_duration: SignIn::Constants::RefreshToken::VALIDITY_LENGTH_SHORT_MINUTES)
+
+# Create Config for example external client
+vatemplate = SignIn::ClientConfig.find_or_initialize_by(client_id: 'vatemplate')
+vatemplate.update!(authentication: SignIn::Constants::Auth::COOKIE,
+                   anti_csrf: true,
+                   redirect_uri: 'http://localhost:3001/auth/login/callback',
+                   access_token_duration: SignIn::Constants::AccessToken::VALIDITY_LENGTH_SHORT_MINUTES,
+                   access_token_audience: 'va.gov',
```

Comment:
I am not super clear on what `access_token_audience` actually means. Is this ok to use for external apps?

Reply:
I think at present it isn't doing anything since SiS doesn't yet have any external clients. But you make a good point, I'll change this to a different value.